### PR TITLE
Nick license readme update

### PR DESCRIPTION
**Pull request by Nick-Harvey**
Updating the License to the new Pachyderm Community License as well as doing a quick Readme refresh.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,190 +1,32 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+Pachyderm License
```
Comment (JoeyZwicker) [inline LICENSE]: Should we call this Community License since that's how we refer to it everywhere else?

File: README.md [hunk 2]
```diff
@@ -45,18 +47,18 @@ If you'd like to see some examples and learn about core use cases for Pachyderm:
...
 ## Community
 Keep up to date and get Pachyderm support via:
-- [![Twitter](https://img.shields.io/twitter/url/https/twitter.com/pachydermio.svg?style=social&label=Follow%20%40pachydermio)](https://twitter.com/pachyderminc) Follow us on Twitter.
+- [![Twitter](https://img.shields.io/twitter/follow/pachyderminc?style=social)](https://twitter.com/pachyderminc) Follow us on Twitter.
 - [![Slack Status](http://slack.pachyderm.io/badge.svg)](http://slack.pachyderm.io) Join our community [Slack Channel](http://slack.pachyderm.io) to get help from the Pachyderm team and other users.
 
-### Contributing
+## Contributing
 
 To get started, sign the [Contributor License Agreement](https://pachyderm.wufoo.com/forms/pachyderm-contributor-license-agreement).
```
Comment (JoeyZwicker) [inline README.md]: Does this wufoo CLA point to the right place? I know we have it living somewhere else for the CLA bot, but idk how these integrate or not? If they dont, we need to update the CLA in both places?

File: README.md [hunk 3]
```diff
@@ -66,3 +68,11 @@ Pachyderm automatically reports anonymized usage metrics. These metrics help us
 understand how people are using Pachyderm and make it better.  They can be
 disabled by setting the env variable `METRICS` to `false` in the pachd
 container.
+
+## License Information
+Pachyderm has moved some components of Pachyderm Platform to a [source-available limited license](LICENSE). 
+We remain committed to the culture of open source, developing our product transparently and collaboratively with our community, giving our community and customers source code access, and the ability to study and change the software to suit their needs.
```
Comment (JoeyZwicker) [inline README.md]:
```suggestion
We remain committed to the culture of open source, developing our product transparently and collaboratively with our community, and giving our community and customers source code access and the ability to study and change the software to suit their needs.
```

fixed the grammar. I think this needs to be changed int eh FAQ too

File: LICENSE (hunk 1 above)
Comment (Nick-Harvey) [reply LICENSE]: I checked everyone I could think of who changed their license:
Mongo - sspl
Redis - 3 clause BSD
Confluent - Confluent Community License Agreement
Sentry -3 clause BSD 
Cockroach - Business Source License 1.1 (interesting Mariadb has actually trademarked this license)
Elastic - ELASTIC LICENSE AGREEMENT
Timescale - Timescale License (TSL)
Neo4j - Commons Clause
Dgraph - Dgraph Community License Agreement
etc.

I don't necessarily see a patter per-say, but I'll change it just to keep things consistent. And just to make sure... the lawyers won't have any issues with us changing the title?

Comment (Nick-Harvey) [reply LICENSE]: fixed in next commit

**Pull request merged**